web/src/components/repo/pipeline/PipelineLog.vue

```diff
@@ -20,6 +20,13 @@
             icon="download"
             @click="download"
           />
+          <IconButton
+            v-if="step?.end_time !== undefined && user?.admin"
+            :title="$t('repo.pipeline.actions.log_delete')"
+            class="!hover:bg-white !hover:bg-opacity-10"
```

Comment:
Do those colors work in light and dark-mode? (Just want to make sure. Seems like you just copied them from above)

Reply:
> you just copied them from above

That is true for all PR :)

> Do those colors work in light and dark-mode?

Yes.
![wp-pp-log-delete-light-dark](https://github.com/woodpecker-ci/woodpecker/assets/127358482/4477a6b6-18b4-4943-ba5a-3cdfddd54614)